language-support/java/bindings-rxjava/BUILD.bazel

```diff
@@ -8,6 +8,18 @@ load(
 )
```

Comment:
please kindly double check changes to this file, I still don't speak bazel :)

Reply:
Don't worry, few people on this planet speak it. I'm at level A1. :laughing: